### PR TITLE
Adds whoopee cushions and makes it easier to add new farts

### PR DESCRIPTION
**Pull request by KoboldCommando**
# About The Pull Request

This adds a whoopee cushion, how in the world did we not have one yet?  It can be bought in the hacked library or theatre vendors, or in the clown costume crate.

image

It also moves all the fart messages to a .txt file so it's effortless to add more!  I added a few myself.

## Why It's Good For The Game

fart funne

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
add: you can now find whoopee cushions in the library and theatre vendors, prank your friends!
code: it's now super easy to add more messages for when people fart!
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1672,3 +1672,21 @@
...
+
+
+/obj/item/toy/whoopee/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/squeak, farts)
+
+/obj/item/toy/whoopee/sord //do not let players acquire this outside of admin shenanigans, for the love of god!
+	desc = "Something seems slightly off about this one..."
+	icon_state = "whoopee_sord"
+	farts = list('monkestation/sound/effects/sordfart14.ogg', 'monkestation/sound/effects/sordfart17.ogg', 'monkestation/sound/effects/sordfart18.ogg', 'monkestation/sound/effects/sordfart20.ogg')
```
Comment (nednaZ) [inline code/game/objects/items/toys.dm:1692]: No.

**Review by nednaZ — CHANGES_REQUESTED** [review]
No Sord farts at all.

**Comment by KoboldCommando** [comment]
sord has been murdered, as commanded

**Review by nednaZ — CHANGES_REQUESTED** [review]
Sordfarts are still in the file changes

**Review by nednaZ — APPROVED** [review]
With the Sord destroyed, this is fine

**Pull request merged**